docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -33,34 +36,98 @@ announced on the ["releases" forum category](https://forum.silverstripe.org/c/re
-silverstripe/cms 4.3.1 with silverstripe/versioned 4.3.9.
+`silverstripe/cms` 4.3.1 with `silverstripe/versioned` 4.3.9.
 
-All SilverStripe recipes are released in lock step with each other. For example, silverstripe/installer 4.3.1 will
-contain silverstripe/recipe-cms 4.3.1 and silverstripe/recipe-core 4.3.1. These recipes may contain various patch
+All Silverstripe CMS recipes are released in lock step with each other. For example, `silverstripe/installer` 4.3.1 will
+contain `silverstripe/recipe-cms` 4.3.1 and `silverstripe/recipe-core` 4.3.1. These recipes may contain various patch
 versions of its modules within the same minor release line (4.3 in this example).
 
+## Regular quarterly releases
+
+A regular Silverstripe CMS minor release is tagged every quarter. This involves creating a new minor increment of 
+each core recipe. Regular quarterly releases are preceded by pre-stable releases to help lay the groundwork for the
+stable release.
+
+### Beta phase
+
+At the start of the Beta phase:
+* new minor branches for core modules are forked from the default branches
+* a feature freeze is instituted on the minor branches
+* a `beta1` release is tagged for recipes and related modules.
+
+The beta phase lasts approximately one month. Additional betas may be released to address critical issues holding back
+testers.
+
+#### "Beta release" definition
+
+The purpose of a beta release is to stabilise the Silverstripe CMS codebase for an upcoming stable release and showcase 
+new features. A beta release is likely to contain regressions and new bugs. A beta release is primarily aimed at testers 
+who want to help identify bugs and regressions.
+
+#### "Feature freeze" definition
+
+A "Feature freeze" is an unreleased minor branch state. This state allows the core team to stabilise the upcoming 
+release and reach production quality.
+
+The following type of changes can be merged in a branch in feature freeze:
+* bug fixes, regardless of severity
+* API changes if they fix a bug and are compliant with our existing semver commitments
+* merge-ups from older minor branches.
+
+The following changes cannot be merged in a branch in feature freeze:
+* any PR that introduces new functionality
+* API changes not targeted to fixing a bug, or that alter an existing API in a stable release
+
+### Release Candidate phase
+
+All critical or high priority issues in the beta release should have been identified and fixed by the start of the
+Release Candidate phase.
+
+At the start of the Release Candidate phase:
+* a publicly visible `rc1` release is tagged for recipes and related modules
+* a private Release Candidate is created containing fixes for undisclosed vulnerabilities on top of the `rc1` release
+* the private Release Candidate is delivered to independent auditors for a security review.
+
+The Release Candidate phase lasts approximately two weeks. Additional patches may be added to the final release only if:
```

Comment:
```suggestion
While the Release Candidate phase lasts additional patches may be added to the final release if:
```

Timeframes are subject to change

Reply:
Same discussion point has the beta period.

Reply:
I disagree with this suggestion. While the beta phase period can be communicated at the time of tagging, the release candidate period has specific dates that are unlikely to change due to our commitments with external auditors. 'Approximately two weeks' seems appropriate to me.